app/models/df_json_wrapper.rb

```diff
@@ -4,32 +4,62 @@ def initialize(json)
     @json = json
   end
 
+  def self.json_accessor(mappings)
+    json_reader(mappings)
+    json_writer(mappings)
+  end
+
   def self.json_reader(mappings)
     mappings.each do |selector_name, selector_info|
-      selector_info => {type:, key:}
+      selector_info => { type:, key: }
+      key_path = key.split
```

Comment:
can you clarify why we need this `key.split`? which keys do we currently have that will have multiple words separated by a whitespace? is this a pre-emptive implementation (i see the tests)

Reply:
yep, I was just trying to have some kind of feature parity with the XML version of this (in `DfXmlAccessor.define_xml_readers`/`DfXmlCrudMethods#df_xml_value`) where spaces can be used in a key and Nokogiri will look it up as a css path in the nested XML structure. I know we're not getting nested objects in the DF JSON currently, but I don't think this is too much speculative complexity to bear, personally.